docs: change default `max_feature_count` documentation to null (#1706)
urbica/martin: docs/src/config-file.md

  # If the source table has more features than set here, they will not be included in the tile and the result will look "cut off"/incomplete.
  # This feature allows to put a maximum latency bound on tiles with extreme amount of detail at the cost of not returning all data.
  # It is sensible to set this limit if you have user generated/untrusted geodata, e.g. a lot of data points at [Null Island](https://en.wikipedia.org/wiki/Null_Island).

>   max_feature_count: max_feature_count: null # null=unlimited (default), but can also be a positive integer

Reviewer: ```suggestion
  max_feature_count: null   # either a positive integer, or null=unlimited (default)
```

Author: Shure, but lets not have so many spaces ^^

```suggestion
  max_feature_count: null # either a positive integer, or null=unlimited (default)
```
